### PR TITLE
Separate annotations out into it's own component

### PR DESCRIPTION
**Pull request by Jaydee123**
### What does this PR do?
I have moved annotations into it's own component so that I am able to test it better as well as make the spectrogram page smaller.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/Features/Annotations/Annotations.test.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import { Annotations } from '@/Features/Annotations/Annotations';
+import React from 'react';
+import metadataJson from './Annotations.test.meta.json';
+
+describe('Annotations feature component', () => {
+  beforeEach(() => {
+    cleanup();
+  });
+  test('Display correct data on initial view', async () => {
+    const metadata = JSON.parse(JSON.stringify(metadataJson));
```
Comment (Nepomuceno) [inline client/src/Features/Annotations/Annotations.test.tsx:14]: This is ugly but it is needed

File: client/src/Features/Annotations/Annotations.test.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  });
+
+  test.each`
+    input                         | label                               | type            | expected
+    ${'883'}                      | ${'Annotation 0 - Frequency Start'} | ${'spinbutton'} | ${'883'}
+    ${'883'}                      | ${'Annotation 0 - Frequency Start'} | ${'spinbutton'} | ${'1.625MHz'}
+    ${'885'}                      | ${'Annotation 0 - Frequency End'}   | ${'spinbutton'} | ${'885'}
+    ${'885'}                      | ${'Annotation 0 - Frequency End'}   | ${'spinbutton'} | ${'1.725MHz'}
+    ${'2022-03-17T16:43:30.002Z'} | ${'Annotation 0 - Start Time'}      | ${'textbox'}    | ${'2022-03-17T16:43:30.002Z'}
+    ${'2022-03-17T16:45:30.003Z'} | ${'Annotation 0 - End Time'}        | ${'textbox'}    | ${'2022-03-17T16:45:30.003Z'}
```
Comment (Nepomuceno) [inline client/src/Features/Annotations/Annotations.test.tsx:65]: Loved the test cases inclusion

File: client/src/Features/Annotations/Annotations.tsx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+  validateDate,
+} from '@/Utils/rfFunctions';
+import AutoSizeInput from '@/Components/AutoSizeInput/AutoSizeInput';
+
+export const Annotations = ({ meta, totalIQSamples, updateSpectrogram }) => {
+  const [metadata, setMetadata] = useState(meta);
+  const [parents, setParents] = useState([]);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
```
Comment (Nepomuceno) [inline client/src/Features/Annotations/Annotations.tsx:20]: is this actually needed because you aready initialize parents ad an empty array.,

**Review by Nepomuceno — APPROVED** [review]
LGTM 

Just a small comment about something that might not be neeedd.

File: client/src/Features/Annotations/Annotations.tsx (hunk 3 above)
Comment (Jaydee123) [reply client/src/Features/Annotations/Annotations.tsx:20]: No! When I was having issues with my tests and added it in as a test and forgot to remove, thanks :D

**Pull request merged**